src/modules/mavlink/streams/GPS_GLOBAL_ORIGIN.hpp

```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2020 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+#pragma once
```

Comment:
I've actually been using guards intentionally for streams (ie `#define GPS_GLOBAL_ORIGIN_HPP`). 

Then the same define used in mavlink_messages to add it to the stream list. https://github.com/PX4/PX4-Autopilot/blob/387659d61589928fbba2d4cf5454eb8f08c89b95/src/modules/mavlink/mavlink_messages.cpp#L4424-L4426

This gives us a mechanism to trivially skip streams at build time by simply not including the header. Something that's likely going to become more important in future flash wars...

https://github.com/PX4/PX4-Autopilot/blob/387659d61589928fbba2d4cf5454eb8f08c89b95/src/modules/mavlink/mavlink_messages.cpp#L130-L135

Reply:
Genious, why did I not get it in the first place 🤦‍♂️

Reply:
Added it 👍

Reply:
Note: It will probably not work with the last element in the list because there will be a comma at the end.